src/autotype/AutoType.cpp

```diff
@@ -632,14 +635,18 @@ AutoType::parseSequence(const QString& entrySequence, const Entry* entry, QStrin
                 error = tr("Very long delay detected, max is %1: %2").arg(maxWaitDelay).arg(fullPlaceholder);
                 return {};
             }
+            cumulativeDelay += repeat;
```

Comment:
Shouldn't this be multiplicative?

Reply:
That's what I figured. But it isn't clear from the name and the regex is pretty opaque. It's not a must, but a comment would help clarify things.